api/src/main/kotlin/org/jetbrains/kotlinx/dl/api/core/util/ContainerUtil.kt

```diff
@@ -20,4 +20,10 @@ public fun <K, V> mapOfNotNull(vararg mapping: Pair<K?, V?>): Map<K, V> {
             }
         }
     }
+}
+
+internal fun Array<IntArray>.deepContentToString(): String {
```

Comment:
Looks like missed thing in the standard library!

Reply:
https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/content-deep-to-string.html